hardware/scripts/dramsys_plot.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
Is this script MemPool specific, or could it also go into the `dram_rtl_sim` dependency and be used for other projects as well?